symphony/lib/toolkit/class.widget.php

```diff
@@ -710,7 +710,7 @@ public static function Select($name, array $options = null, array $attributes =
      */
     private static function __SelectBuildOption($option)
     {
-        @list($value, $selected, $desc, $class, $id, $attributes) = $option;
+        list($value, $selected, $desc, $class, $id, $attributes) = array_pad($option, 6, null);
```

Comment:
It's ok to remove the silencer here ? What if the list call still fails somehow ?